clientv3/balancer.go

```diff
@@ -131,6 +144,51 @@ func (b *simpleBalancer) updateAddrs(eps []string) {
 	}
 }
 
+func (b *simpleBalancer) updateNotifyLoop() {
+	defer close(b.donec)
+
+	var downc chan struct{}
+	for {
+		b.mu.RLock()
+		upc := b.upc
+		b.mu.RUnlock()
+		select {
+		case <-upc:
+			var addr string
+			b.mu.RLock()
+			addr = b.pinAddr
+			// Up() updates upc and downc at the same time;
+			// retrieve downc immediately.
```

Comment:
This comment is misleading since "immediately" implies there's some kind of synchronization going on, when there isn't (there can be a up/down cycle between receiving `<-upc` and acquiring the rlock), maybe `// Up() sets pinAddr and downc as a pair under b.mu`?